### ISSUE TITLE
Drafts

### ISSUE THREAD

**Issue by jnordberg** [steemit/steemitapi#2]
Add rpc methods to list/add/remove drafts, saved in s3

Data structure draft for drafts:
```javascript
[
  {
    title: "draft post title",
    body: "I'll do it live damnit!",
    date: "2017-09-19T19:37:12.917Z",
    uuid: "3A60740F-06AC-4BE4-B017-96150DE1510F"
  }
  ..
]
```

Stored under bucket/steemitapit_drafts_<account>.json

**Comment by sneak** [comment]
looks great! suggest filename `bucket/steemitapi_#{username}_drafts.json`

**Comment by jnordberg** [comment]
Implemented in #4
